test/models/transfer_test.rb

```diff
@@ -39,6 +39,59 @@ def teardown; end
     assert_includes(exception.message, 'Who')
   end
 
+  test 'update patient linelist' do
+    # Create new patient and verify default linelist values
```

Comment:
hmm my reasoning for putting it here is because even though we are modifying the patient model from here, the actual method we're testing is in the transfer model

Reply:
my previous comment was based on an incorrect interpretation, it is addressed here now: https://github.com/SaraAlert/SaraAlert/commit/7115288aed25e17c0dde8da723620603f97055ae#diff-d15cb16c70952baabc3701f60d192aa6R11